docs/migration/migration.adoc

```diff
@@ -1,2 +1,9 @@
 = Migrációs leírások
 
+== 1.1.0
```

Comment:
Ide nem soroljuk fel a pluginokat tételesen? Meg a jandexet is?

Reply:
Tételesen nincs értelme felsorolni, de a jandex plugint beleírtam